test/unittests/api/v0.0.1/Candidates_Test.php

```diff
@@ -97,17 +96,52 @@ function testPostCandidateValid() {
                     ]
                 ]
                 );
+            return $API;
         } catch(\Loris\API\SafeExitException $e) {
             $API = $e->Object;
+            return $API;
         }
+    }
+    function testPostCandidateValid() {
```

Comment:
These changes to the test don't test that a validate post creates a candidate, they test that a user with an unsupported number of centerIDs *can't* create a candidate. It's *removed* the test case for checking that valid posts create candidates, which is the most important thing to ensure works.